project/profiles/capacity4more/modules/c4m/content/c4m_content_photoalbum/c4m_content_photoalbum.module

```diff
@@ -23,6 +23,39 @@ function c4m_content_photoalbum_c4m_og_vocab_info_content() {
 }
 
 /**
+ * Implements hook_form_FORM_ID_alter().
+ *
+ * Adds an `Add Photos` button to the `Create Photo album` form.
+ */
+function c4m_content_photoalbum_form_photoalbum_node_form_alter(&$form, &$form_instance, $form_id) {
+  $form['actions']['add_photos'] = array(
```

Comment:
@amitaibu 
It seems to be the request here:
https://issuetracker.amplexor.com/jira/browse/CFM-1211

<img width="1146" alt="_cfm-1211__update_photo_and_photoalbum_content_type_-_amplexor" src="https://cloud.githubusercontent.com/assets/5985683/16261506/2a54b2f6-386c-11e6-99dd-904389d976f4.png">